custom_components/mqtt_discoverystream/__init__.py

```diff
@@ -6,7 +6,7 @@
 import voluptuous as vol
 
 from homeassistant.components.mqtt import (
-  valid_publish_topic, 
+  valid_publish_topic,
```

Comment:
Sorry for the line break / empty space cleanups. VSCode is very particular about cleaning them up on save.